src/sync.cpp

```diff
@@ -73,7 +73,11 @@ struct LockData {
     LockOrders lockorders;
     InvLockOrders invlockorders;
     std::mutex dd_mutex;
-} static lockdata;
+};
+LockData& GetLockData() {
```

Comment:
@promag Thanks for your reviewing.
Does this mean that the scope of ```GetLockData()``` should be compile-unit local same as original ```lockdata``` to prevent external calling?

Reply:
Yes linkage changed, but I don't think it warrants a PR.